### PR TITLE
DroneCAN: make himark servos an optional build item

### PR DESCRIPTION
**Pull request by tridge**
this came out of this PR for 4.4.1:
https://github.com/ArduPilot/ardupilot/pull/24713

### CODE REVIEW

File: Tools/scripts/build_options.py [hunk 1]
```diff
@@ -335,6 +335,8 @@ def __init__(self,
     Feature('Actuators', 'SBUS Output', 'AP_SBUSOUTPUT_ENABLED', 'Enable SBUS Output on serial ports', 0, None),
     Feature('Actuators', 'FETTecOneWire', 'AP_FETTEC_ONEWIRE_ENABLED', 'Enable FETTec OneWire ESCs', 0, None),
     Feature('Actuators', 'KDECAN', 'AP_KDECAN_ENABLED', 'KDE Direct KDECAN ESC', 0, None),
+    Feature('Actuators', 'HimarkServo', 'AP_DRONECAN_HIMARK_SERVO_SUPPORT', 'Enable Himark DroneCAN servos', 0, None),
```
Comment (peterbarker) [inline Tools/scripts/build_options.py:338]:
```suggestion
    Feature('Actuators', 'HimarkServo', 'AP_DRONECAN_HIMARK_SERVO_ENABLED', 'Enable Himark DroneCAN servos', 0, None),
```

File: Tools/scripts/build_options.py [hunk 2]
```diff
@@ -335,6 +335,8 @@ def __init__(self,
...
+    Feature('Actuators', 'HobbywingESC', 'AP_DRONECAN_HOBBYWING_ESC_SUPPORT', 'Enable Hobbywing DroneCAN ESCs', 0, None),
```
Comment (peterbarker) [inline Tools/scripts/build_options.py:339]:
```suggestion
    Feature('Actuators', 'HobbywingESC', 'AP_DRONECAN_HOBBYWING_ESC_ENABLED', 'Enable Hobbywing DroneCAN ESCs', 0, None),
```

File: Tools/scripts/extract_features.py [hunk 3]
```diff
@@ -212,6 +212,9 @@ def __init__(self, filename, nm="arm-none-eabi-nm"):
             ('AP_MAVLINK_BATTERY2_ENABLED', 'GCS_MAVLINK::send_battery2'),
             ('AP_MAVLINK_MSG_MOUNT_CONTROL_ENABLED', 'AP_Mount::handle_mount_control'),
             ('AP_MAVLINK_MSG_MOUNT_CONFIGURE_ENABLED', 'AP_Mount::handle_mount_configure'),
+
+            ('AP_DRONECAN_HIMARK_SERVO_SUPPORT', 'AP_DroneCAN::SRV_send_himark'),
+            ('AP_DRONECAN_HOBBYWING_ESC_SUPPORT', 'AP_DroneCAN::hobbywing_ESC_update()'),
```
Comment (peterbarker) [inline Tools/scripts/extract_features.py:217]:
```suggestion
            ('AP_DRONECAN_HOBBYWING_ESC_SUPPORT', 'AP_DroneCAN::hobbywing_ESC_update'),
```

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: Tools/scripts/build_options.py (hunk 1 above)
Comment (tridge) [reply Tools/scripts/build_options.py]: done

File: Tools/scripts/extract_features.py (hunk 3 above)
Comment (tridge) [reply Tools/scripts/extract_features.py]: done

**Review by peterbarker — APPROVED** [review]
LGTM

**Pull request merged**